### PR TITLE
replace screenshot with code block

### PR DESCRIPTION
**Pull request by mirnawong1**
replacing screenshot with code block to improve accessibility and ux.

also removed image file

### CODE REVIEW

File: website/docs/reference/commands/compile.md [hunk 1]
```diff
@@ -33,13 +33,42 @@ dbt compile --select stg_payments
 dbt compile --inline "select * from {{ ref('raw_orders') }}"
 ```
 
-<Lightbox src="/img/docs/reference/dbt-compile.png" title="dbt compile --select stg_payments"/>
+returns the following:
+
+
+```bash
+dbt compile --select stg_orders
```
Comment (mirnawong1) [inline website/docs/reference/commands/compile.md:40]: should this be ` --select stg_payments`? instead per example above this section? 

```
dbt compile --select stg_payments
dbt compile --inline "select * from {{ ref('raw_orders') }}"

```

Comment (mirnawong1) [reply website/docs/reference/commands/compile.md:40]: so should the example change to `stg_orders`?

Comment (joellabes) [reply website/docs/reference/commands/compile.md:40]: Having the basic demo and the worked example reference the same things isn't the worst idea! Feel free to change it 👍

**Review by joellabes — APPROVED** [review]

**Comment by mirnawong1** [comment]
thanks for the approval @joellabes ! i think i understand it a bit more 🙏

**Pull request merged**